### PR TITLE
70 date time picker

### PR DESCRIPTION
**Pull request by ntibaldi92**
This PR replaces the `material-ui` date and time picker with `React-Kronos.`

As of now, all of the meeting form's validations/behavior remain the same, but Kronos allows for us to:
- customize the date/time formats
- prevent the user from clicking on dates in the past
- prevent display of times in the past
- customize the meeting time intervals

This is much more user-friendly and less confusing.

The date and time `input` fields in Kronos are easily customized via the component's props. Unfortunately, some hacky CSS magic in the `index.ejs` was required to make the datepicker's calendar and timepicker's dropdown look consistent with the designs for Bookit.

### CODE REVIEW

**Comment by ntibaldi92** [comment]
Uhhh... #70 was the ticket number for the SPIKE, this PR is for ticket #93, sorry guyz for the bad naming.

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ env:
 
 install:
   - if [[ $TRAVIS_PULL_REQUEST == "false" ]] && [[ $TRAVIS_BRANCH == "master" ]]; then pip install awscli; fi
+  - yarn
```
Comment (BillyZac) [inline .travis.yml]: Is this related to the date/time picker?
